VideoLocker/res/values/strings.xml

```diff
@@ -477,6 +477,14 @@
     <!-- In Sign up screen -->
     <string name="or_sign_up_with_email">or sign up with email</string>
 
+    <!-- New Logistration Launch screen text -->
+    <!-- Button leading user to create account screen -->
+    <string name="launch_register">Register</string>
+    <!-- Button leading user to Find Courses screen from the launch screen -->
+    <string name="launch_discover_courses">@string/find_courses_title</string>
```

Comment:
@catong @marcomorales The "Find Courses" string is a resource, if we change it to "Discover Courses", all the instances of "Find Courses" will be changed to "Discover Courses" as well. I don't mind this change either way, but there will be another inconsistency "Find a mobile-friendly course" string at the bottom of MyCourses. 

I guess for now, we should create a new string resource for the button and make the single change.